help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -147,12 +139,23 @@ In the **[!UICONTROL Source field]**, you can select any attribute or identity f
 ![map desired data fields to the device_id field](../../assets/catalog/advertising/magnite/destination-batch-active-audience-field-mapping.png)
 
 In the **[!UICONTROL Target field]**:
-![select the appropriate device type target identity](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-device-type.png) See [Supported Identities](#supported-identities) for more information.
-In this example, we've selected the **[!UICONTROL Target field]**: magnite_deviceId_CUSTOM, because our **[!UICONTROL Source field]** was defined as a custom IdentityMap: DeviceID. 
+The next step is mapping Source Field identifiers to the [!DNL Magnite] Device Type identifier of your choice, located in the Target Field.
+
+The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.
+
+In order to support a dynamic list of device types, the dropdown will provide a list of integers (from 0-50), each of which represents a specific device type in [!DNL Magnite]'s system. To understand which device type id in the list of integers corresponds to which [!DNL Magnite] device type, reach out to your [!DNL Magnite] account representative.
+
+![Choose desired device type integer attribute from target field dropdown](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-device-type.png)
+
+See [Supported Identities](#supported-identities) for more information.
+In this example, we've selected the **[!UICONTROL Target field]**: Attribute: "00".
+!["Attribute: 00" was selected](../../assets/catalog/advertising/magnite/destination-batch-active-audience-field-mapping.png)
+
+* You can add as many mappings as you want/need by selecting **[!UICONTROL Add new mapping]**.
 
 >[!NOTE]
 >
->If you plan to send/map multiple ID types (GAID, IDFA, etc.) using the Batch destination, a new destination connection/instance is required for each. Please contact your Magnite Account representative for more information.
+>If you plan to map an audience to multiple device types (GAID, IDFA, etc.) using the Batch destination, a new destination connection/instance is required for each mapping. Please contact your [!DNL Magnite] account representative for more information.
```

Comment:
"Please contact your [!DNL Magnite] account representative for more information."

What would they contact Magnite for? Is there anything else to do aside from creating a new connection? Anything that Magnite needs to do on their side?